src/main/java/org/breedinginsight/api/v1/controller/BreedingMethodController.java

```diff
@@ -64,7 +59,7 @@ public HttpResponse<Response<DataResponse<ProgramBreedingMethodEntity>>> getSyst
     @Post("programs/{programId}/breeding-methods")
     @Produces(MediaType.APPLICATION_JSON)
     @ProgramSecured(roles = {ProgramSecuredRole.BREEDER})
-    public HttpResponse<Response<ProgramBreedingMethodEntity>> createProgramBreedingMethod(@PathVariable UUID programId, @Body ProgramBreedingMethodEntity breedingMethod) throws BadRequestException, ApiException {
+    public HttpResponse createProgramBreedingMethod(@PathVariable UUID programId, @Body ProgramBreedingMethodEntity breedingMethod) throws ApiException {
```

Comment:
Is it more idiomatic to return `HttpResponse<Object>`? Just trying to make the linter happy.

Reply:
Probably better to return `HttpResponse<?>` rather than `HttpResponse<Object>`

Reply:
Thanks. Done.